### PR TITLE
fix(fa): Fix attachments placement when printing application

### PR DESCRIPTION
**Pull request by davidpxn**
# ... 👆

https://app.asana.com/0/1202037685216853/1202070037203638

## What

Fix styling of the print page.

## Why

So the attachments don't overlap with the timeline component.

## Screenshots / Gifs

Before: 
image

After:
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-veita/src/styles.css [hunk 1]
```diff
@@ -212,7 +212,6 @@ input[type='number'] {
   .printableSection {
     float: left;
     width: 100vw;
-    height: 100vh;
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-veita/src/styles.css:215]: why are we removing this? 🤔

File: apps/financial-aid/web-veita/src/routes/PrintableApplication/printableApplication.tsx [hunk 2]
```diff
@@ -47,9 +48,9 @@ const PrintableApplication = () => {
             isPrint={true}
           />
           {application?.files && (
-            <>
+            <Box marginTop={20}>
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-veita/src/routes/PrintableApplication/printableApplication.tsx:51]: We could add the margin when print under css print, should we maybe do that instead?

**Review by MargretFinnboga — COMMENTED** [review]
questions 💭

File: apps/financial-aid/web-veita/src/routes/PrintableApplication/printableApplication.tsx (hunk 2 above)
Comment (davidpxn) [reply apps/financial-aid/web-veita/src/routes/PrintableApplication/printableApplication.tsx:51]: I decided to have it here to create a bit of a margin between the timeline and the attachments. 
So before it looked like this:
image

Now its like this: 
image

**Pull request merged**